vagga.yaml

```diff
@@ -98,8 +98,12 @@ commands:
   test: !Command
     description: Run tests
     container: test
+    run: python3 manage.py test
```

Comment:
I would add some verbosity like `-v` or `-vv`, so that the list of tests will be available in the CI build log.